index.html

```diff
@@ -116,7 +133,7 @@ <h2>Project name goes here</h2>
                 <p>Ruby on Rails</p>
             </div>
             <div class="javascript-rectangle">
-                <p>Javascript</p>
+                <p class="js-text">Javascript</p>
             </div>
             <div class="see-project">
                 <h3>See this project</h3>
```

Comment:
- [ ] `See this project` is a button, therefore, should be inside a `<button>` or `<a>` tag. Buttons on a dark background should have light background and dark text:

Your design  | Figma design
------------- | -------------
<img width="426" alt="Screenshot 2022-03-31 at 16 38 51" src="https://user-images.githubusercontent.com/77038610/161056578-173f9aa4-9c00-4bdf-be10-91016753a03e.png"> | <img width="370" alt="Screenshot 2022-03-31 at 16 39 14" src="https://user-images.githubusercontent.com/77038610/161056638-7c49d971-d6ff-4e7c-942e-3c378f0aac0f.png">

Reply:
Fixed this section as well 
<img width="963" alt="Screenshot 2022-04-01 at 08 15 36" src="https://user-images.githubusercontent.com/40231363/161199349-b1381f95-ec2a-4a66-80ab-60d116caa7db.png">